src/atom_finder/classifier.clj

```diff
@@ -30,16 +35,21 @@
   [name classifier finder]
   `(Atom. (s/validate AtomName ~name)
           (s/fn ~(symbol (str name "-classifier")) :- Boolean [node# :- IASTNode] (~classifier node#))
-          (s/fn ~(symbol (str name "-finder")) :- [IASTNode] [node# :- IASTNode] (~finder node#))
-  ))
+          (s/fn ~(symbol (str name "-finder")) :- [IASTNode] [node# :- IASTNode] (~finder node#))))
```

Comment:
Can you please check if there's a way to turn off auto-formatting of lines you don't edit?